functions.go

```diff
@@ -24,6 +24,10 @@ func (e ErrInvalidFunctionCall) Error() string {
 	return "Invalid function call: " + e.Msg
 }
 
+func RegisterGlobalFuncs(def string, fn Function) {
```

Comment:
Rename to RegisterGlobalFunc, and no need to get `def`, use fn.Name, also make it RegisterGlobalFunc(fn...Function)